src/helpers/gutter/gutter.js

```diff
@@ -9,11 +9,11 @@ import { themeProvider } from '../../theme';
 const { theme } = themeProvider;
 
 export const gutterWidth = (props, breakpoint) => {
+  if (has(props, `gutter.${breakpoint}`)) {
```

Comment:
Sure. I'll refactor what I have, might find I end up in the same spot with needing to check if the prop exists first and if its an object.